app/Http/Resources/AuctionResource.php

```diff
@@ -0,0 +1,32 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Otoszroto\Http\Resources;
+
+use Illuminate\Http\Request;
+use Illuminate\Http\Resources\Json\JsonResource;
+
+class AuctionResource extends JsonResource
+{
+    /**
+     * @return array<string, mixed>
+     */
+    public function toArray(Request $request): array
+    {
+        return [
+            "id" => $this->id,
+            "name" => $this->name,
+            "description" => $this->description,
+            "photo_url" => $this->photo_url,
+            "price" => $this->price,
+            "owner_id" => $this->owner_id,
+            "model_id" => $this->model_id,
+            "category_id" => $this->category_id,
+            "condition_id" => $this->condition_id,
+            "auction_state" => $this->auction_state,
+            "created_at" => $this->created_at,
+            "updated_at" => $this->updated_at,
```

Comment:
```suggestion
        return [
            "id" => $this->id,
            "name" => $this->name,
            "description" => $this->description,
            "photo" => $this->photo_url,
            "price" => $this->price,
            "owner" => UserResource::make($this->owner),
            "model" => $this->model->name,
            "category" => $this->category->name,
            "condition" => $this->condition,
            "auctionState" => $this->auction_state,
            "createdAt" => $this->created_at,
            "updatedAt" => $this->updated_at,
```

Ułatwmy naszemu biednemu frontasiowi życie, to też człowiek. Niech wysyła zapytanie raz i otrzymuje od razu wszystkie dane. Mając `id` usera muszę wysyłać kolejne zapytanie o jego nazwę, nie potrzebnie obciążając serwer kolejnym zapytaniem SQL. Lepiej na raz od razu wysłać wszystkie i tak potrzebne na froncie dane. Przy okazji fajnie jest je zwracać w camelCase.